src/main/java/qna/domain/Answer.java

```diff
@@ -3,14 +3,25 @@
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
+import javax.persistence.*;
 import java.util.Objects;
 
-public class Answer {
+@Entity
+public class Answer extends AuditingEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private Long writerId;
-    private Long questionId;
+    @Lob
     private String contents;
-    private boolean deleted = false;
+
+    @Column(nullable = false)
+    private Boolean deleted = false;
+    private Long questionId;
+
+    private Long writerId;
```

Comment:
```suggestion
    @Column(nullable = false)
    private Boolean deleted = false;
    private Long questionId;
    private Long writerId;
```

다른 코드와 통일성을 지켜보면 어떨까요?